src/components/ErrorBoundary/DefaultError.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from 'react';
+import ErrorMask from 'box-react-ui/lib/components/error-mask/ErrorMask';
+import { FormattedMessage } from 'react-intl';
+import messages from '../messages';
+import './defaultError.scss';
+
+const DefaultError = () => (
+    <section className="be-default-error">
+        <ErrorMask
```

Comment:
BTW, I haven't seen ErrorMask used at other places, and when we had it for the sidebar before, product/design made me remove it in lieu of just a red error. Not sure whats the status of this component.

Reply:
Checking on which component we should use